main.py

```diff
@@ -14,12 +14,14 @@ def reEmail(fname):
   
   fhand = open(fname,'r')
   for line in fhand:
-    extr = re.findall('.*',line)
+    extr = re.findall('\S*@\S*',line)
```

Comment:
Your regex will extract ALL the characters to the left and right of the @ symbol.
`extr = re.findall('\S+@\S+',line)`

Reply:
COUNT is missing.